lib/trino-hdfs/src/main/java/io/trino/filesystem/hdfs/HdfsTrinoInputStream.java

```diff
@@ -107,6 +107,19 @@ public int read(byte[] b, int off, int len)
         }
     }
 
+    @Override
+    public long skip(long n)
+            throws IOException
+    {
+        ensureOpen();
+        try {
+            return stream.skip(n);
```

Comment:
I probably need to implement `TrinoS3InputStream.skip` - `TestHdfsFileSystemS3Mock.testInputFile` is failing.